### ISSUE TITLE
I am wondering how to change the CRF layer to Fuzzy or Partial CRF

### ISSUE THREAD

**Issue by nooralahzadeh** [flairNLP/flair#532]
A clear and concise description of what you want to know.
Any advice to update the CRF layer ( loss function ) to be as fuzzy CRF. the difference is you have to calculate numerator in a different way ( since you have some known target and unknown target)  like the following link: <https://stackoverflow.com/questions/54575275/partial-crf-in-pytorch?noredirect=1#comment95995257_54575275>
